app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Nominations::NominateInductionCoordinatorController < ApplicationController
+  def new
+    @token = params[:token]
+    nomination_email = NominationEmail.find_by(token: @token)
```

Comment:
This feels a bit as staroing plain-text password. Should we bcrypt the token?

Reply:
I take your point, but I'm happy to leave this as is for now, since it's ephemeral. @cfq any thoughts?

Reply:
Whatever we do here we should also do with magiclink sign in tokens

Reply:
I think there's also the possibility of timing attacks, since find_by isn't constant time.

Reply:
Yes, and only in dev environments.

Reply:
Ah, no definitely should be tackled, if at all, in another PR